README.md

```diff
@@ -1 +1,4 @@
 # programmers-e-commerce
+
```

Comment:
I didn't want to write our names as code owners. Didn't know if you were fine with that. What do you say?

Also did I miss anything else?